plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/controls/CustomTimeEditor.java

```diff
@@ -1,6 +1,6 @@
 /*
  * DBeaver - Universal Database Manager
- * Copyright (C) 2010-2021 DBeaver Corp and others
+ * Copyright (C) 2010-2017 Serge Rider (serge@jkiss.org)
```

Comment:
Copyright (C) 2010-2021 DBeaver Corp and others

Reply:
Fixed